Sources/TuistGraph/DependenciesGraph/DependenciesGraph.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import TSCBasic
+
+/// A directed acyclic graph (DAG) that Tuist uses to represent the third party dependency tree.
```

Comment:
Right now this does not represent a DAG - but creating a DAG will be necessary for SPM, so feel free to keep it (we don't create edges between Carthage's dependencies as they are not defined)